project/plugins.sbt

```diff
@@ -12,6 +12,7 @@ addSbtPlugin("io.gatling"         %  "gatling-sbt"           % "2.1.5")
 addSbtPlugin("io.get-coursier"    %  "sbt-coursier"          % "1.0.0-RC1")
 addSbtPlugin("io.spray"           %  "sbt-revolver"          % "0.8.0")
 addSbtPlugin("io.verizon.build"   %  "sbt-rig"               % "2.0.29")
+addSbtPlugin("org.lyranthe.sbt"   %  "partial-unification"   % "1.1.0")
```

Comment:
No.  I _think_ our build will work without this in master.  It hides a lot of ugliness in 0.16.